docs/website/docs/plus/production/prefect-integration.md

```diff
@@ -0,0 +1,246 @@
+---
+title: "Prefect Integration"
+description: Run pipelines with Prefect
+keywords: ["prefect", "retry", "logs", "artifacts", "progress", "callbacks", "source decomposition", "parallelization"]
+---
+# Prefect Integration
+
+dlt+ offers a few tools and helpers to make running dlt pipelines in prefect a seamless experience.
+
+## Key features
+
+- **Prefect Collector:** a dedicated way to do real-time [progress monitoring] and summary reports after each pipeline stage
```

Comment:
why `[progress monitoring]` is in braces?

Reply:
thx, i thought maybe this could be a link but removed it ultimately: https://dlthub.com/docs/general-usage/pipeline#monitor-the-loading-progress